sessionctx/variable/sysvar_test.go

```diff
@@ -674,6 +674,32 @@ func TestDefaultMemoryDebugModeValue(t *testing.T) {
 	require.Equal(t, val, "0")
 }
 
+func TestSetTIDBDistributeReorg(t *testing.T) {
+	vars := NewSessionVars(nil)
+	mock := NewMockGlobalAccessor4Tests()
+	mock.SessionVars = vars
+	vars.GlobalVarsAccessor = mock
+
+	distributeReorg := GetSysVar(TiDBDDLEnableDistributeReorg)
+
+	// Default off
+	require.Equal(t, distributeReorg.Value, Off)
```

Comment:
If you use feature tag. The default value will be changed.

Reply:
Updated